app/src/context/state.tsx

```diff
@@ -0,0 +1,61 @@
+import clinics from '@public/clinic-output/clinics-with-ids.json'
```

Comment:
Not for addressing in this PR:

I don't think we need to use context in our app. I think we can pass the session object directly to the `<Component>` in _app.tsx. Let's leave it for now, but I wanted to share that I think we can refactor it out in the future.